lib/form1010_ezr/service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'hca/enrollment_system'
+require 'hca/configuration'
+require 'va_1010_forms/service_utils'
+
+module Form1010Ezr
+  class Service < Common::Client::Base
+    include VA1010Forms::ServiceUtils
+
+    # Due to using the same endpoint as the 10-10EZ (HealthCareApplication), we
+    # can utilize the same client configuration
+    configuration HCA::Configuration
+
+    FORM_ID = '10-10EZR'
+
+    # @param [Object] user
+    def initialize(user)
+      super()
```

Comment:
this line is not needed

Reply:
I tried to leave it out, but Rubocop complained when I did:
![](https://github.com/department-of-veterans-affairs/vets-api/assets/34111449/fb08a821-0d44-4e70-bda0-80b20a3c6ea9)